internal/controller/pipelinegate_controller.go

```diff
@@ -124,19 +173,7 @@ func (r *PipelineGateReconciler) reconcilePendingGate(ctx context.Context, gate
 		gateState := v1alpha1.GateState(console.GateStatePending)
 		gate.Status.State = &gateState
 		gate.Status.JobRef = &jobRef
-		if err := r.Status().Update(ctx, gate); err != nil {
-			log.Error(err, "Failed to update PipelineGate status at CR.")
-			return ctrl.Result{}, err
-		}
-
-		// try to update gate at console
-		attributeState := console.GateStatePending
-		updateAttributes := console.GateUpdateAttributes{State: &attributeState, Status: &console.GateStatusAttributes{JobRef: gate.Status.JobRef}}
-		if err := r.ConsoleClient.UpdateGate(gate.Spec.ID, updateAttributes); err != nil {
-			log.Error(err, "Failed to update PipelineGate status to console")
-		}
 
-		log.V(1).Info("Created new job for gate and updated status.", "Name", gate.Name, "ID", gate.Spec.ID, "State", *gate.Status.State, "jobRef", *gate.Status.JobRef)
 		return ctrl.Result{}, nil
 	} else {
```

Comment:
I think this else block can actually be removed since you always return in the above clause (and can save some indentation as a result)